.yamllint.yml

```diff
@@ -12,4 +12,6 @@ rules:
   comments-indentation:
     ignore: .circleci/config.yml
 
-ignore: scripts/aws/aws-k8s-cni.yaml
+ignore: |
+  scripts/aws/aws-k8s-cni.yaml
+  deployments/helm/*/templates/_*.yaml
```

Comment:
Why do we need to ignore this?

Reply:
Because I couldn't find a way to overcome linter error:
 `
./deployments/helm/nsm/templates/_helpers.yaml
  3:1       error    syntax error: expected '<document start>', but found '{'
`

Reply:
Just change its extension to `.tpl` :)

Reply:
- [ ] change its extension to .tpl